doc/source/_static/css/kos_theme.css

```diff
@@ -0,0 +1,26 @@
+/* start from read-the-docs' standard theme: */
+@import url("theme.css");
+
+/* Some of the syntax highlighing done by
+ * read-the-docs is barely visible unless the
+ * font is zoomed large.  At 10 point or 12 point,
+ * boldface doesn't stand out on the font they use,
+ * and many of the keywords are only highlighted
+ * by being bold.  So doing a slight tint from 
+ * the default black just to make it stand out a
+ * little bit more:
+ */
+.k { color:#042; } /*keywords*/
+.o { color:#024; } /*operators*/
+
+/* :struct: links were appearing as black text and didn't look
+ * at all like links.  This says that any border element put
+ * inside a link should share the link's blue color, thus
+ * the :struct: boxes are blue'ed when they are in a link, and
+ * default (black) when they are not links.
+ */
+.a { border-color: #2980B9; }
+
```

Comment:
Adding the following line will highlight structure references (":struct:" tags) blue to match links:
```
a.reference span.pre { color:#2980B9; }
```